### ISSUE TITLE
Sign-In via OIDC not proceeding on the first try.

### ISSUE THREAD

**Issue by vktrrdk** [hashicorp/vault#17022]
**Describe the bug**

When trying to sign in to Vault via OIDC the corresponding pop-up window appears and the authentication with the given provider can be performed. Now the notification shows up which informs, that the sign in proceeds, and that it will close automatically. Nevertheless the window does not close and the login does not successfully proceed. After reloading of the page the same procedure succeeds. 

**To Reproduce**
1. Sign in to Vault via OIDC.
2. Enter your data from your given OICD provider.
3. Window does not automatically close, sign-in does not succeed in first try. 

**Expected behavior**
Instead of needing to reload the page the login should succeed on the first try. 

**Environment:**
* Vault Server Version (retrieve with `vault status`): Version 1.11.3
* Vault CLI Version (retrieve with `vault version`): Vault v1.11.3 (17250b25303c6418c283c95b1d5a9c9f16174fe8), built 2022-08-26T10:27:10Z
* Server Operating System/Architecture: Docker-Container with image vault:1.11.3 running on Ubuntu 20.04.5 LTS.

Vault server configuration file(s):

```hcl
storage "file" {
  path = "/vault/files"
}
listener "tcp" {
  address     = "0.0.0.0:8200"
  tls_disable = 1
}

```

**Additional context**

This behavior shows both on our production and staging environment - both running the same version. 

![image](https://user-images.githubusercontent.com/43116821/188671472-8abcfccb-b4a0-475a-b66b-16cd2e490519.png)

**Comment by macmiranda** [comment]
We face a similar issue but instead the browser closes the pop-up ahead of time and the authentication fails as well... some sort of race condition. 
It normally works on the second or third try.

**Comment by hellobontempo** [comment]
Thank you for reporting @vktrrdk - have you run into this bug since [19071](https://github.com/hashicorp/vault/pull/19071) was merged?

**Comment by macmiranda** [comment]
Yes, still happening with the latest version although not as often

**Comment by dennisme** [comment]
Hey yep we are running 1.14.1 and seeing the issue. In the CLI flow the pop up is a full window/tab and we don't see the timing issue. Is there a way to configure it to use a tab instead of a pop up when logging in via the UI?

**Comment by hsimon-hashicorp** [comment]
Hi folks! Is this still an issue in newer versions of Vault? Please let me know so I can bubble it up accordingly. Thanks!

**Comment by voiprodrigo** [comment]
Hi. It sure is. My team complains all the time, they don't let me forget about it. Using latest release.

**Comment by hsimon-hashicorp** [comment]
> Hi. It sure is. My team complains all the time, they don't let me forget about it. Using latest release.

Thank you for the quick reply! I'll update the issue accordingly.. :)

**Comment by heatherezell** [comment]
Thanks for the report! This issue was filed against Vault `v1.11.3`, which is no longer a supported release.

We're unable to investigate bugs against unsupported versions. If you're still experiencing this on a current supported release of Vault, please open a new issue with updated environment details and we'll take a look.

For a list of currently supported versions, see the [Vault version support page](https://github.com/hashicorp/vault/blob/main/.release/versions.hcl). For Enterprise long-term support information, see the [LTS tracker](https://developer.hashicorp.com/vault/docs/updates/lts-tracker).
